src/std/io/file.ss

```diff
@@ -71,6 +71,13 @@
         (set! self.closed? #t)
         (close-port self.fd)))))
 
+(defmethod {seek file-io}
+  (lambda (self position (from 'start))
```

Comment:
no need for from to be optional, which slows things down; the argument is resolved at the interface, so just a plain argument!